classes/local/store/s3/client.php

```diff
@@ -292,35 +292,33 @@ protected function get_exception_details($exception) {
         return $details;
     }
 
-    public function define_amazon_s3_check($mform, $testdelete = true) {
-        global $OUTPUT;
-        $connection = false;
-
+    public function define_amazon_s3_check($testdelete = true) {
+        global $SESSION;
+        $SESSION->notifications = [];
         $connection = $this->test_connection();
-
         if ($connection->success) {
-            $mform->addElement('html', $OUTPUT->notification($connection->message, 'notifysuccess'));
-
+            \core\notification::success($connection->message);
             // Check permissions if we can connect.
             $permissions = $this->test_permissions($testdelete);
             if ($permissions->success) {
-                $mform->addElement('html', $OUTPUT->notification(key($permissions->messages), current($permissions->messages)));
+                \core\notification::success(key($permissions->messages), current($permissions->messages));
             } else {
                 foreach ($permissions->messages as $message => $type) {
-                    $mform->addElement('html', $OUTPUT->notification($message, $type));
+                    \core\notification::warning($message, $type);
                 }
             }
         } else {
-            $mform->addElement('html', $OUTPUT->notification($connection->message, 'notifyproblem'));
-            $permissions = false;
+            \core\notification::error($connection->message);
         }
-        return $mform;
     }
 
-    public function define_client_section($mform, $config) {
 
-        $mform->addElement('header', 'awsheader', get_string('settings:aws:header', 'tool_objectfs'));
-        $mform->setExpanded('awsheader');
+    /**
+     * @param $settings admin_settingpage
```

Comment:
suggested syntax for the `param` tag:
- `@param [Type] [name] [<description>]`

Reply:
Resolved in 37702e7